src/merkle/empty_nodes.rs

```diff
@@ -0,0 +1,649 @@
+use super::{Felt, RpoDigest, WORD_SIZE, ZERO};
+use core::slice;
+
+// EMPTY NODES SUBTREES
+// ================================================================================================
+
+/// An empty sub-trees set.
+///
+/// Will provide precomputed node values for path computation for pre-defined depths.
+pub struct EmptyNodesSubtrees;
```

Comment:
nit: I think of these more like roots of empty subtrees. So, maybe we should change the name to something like `EmptySubtreeRoots`?

If we do this, the second line of the comment could be: "Contains precomputed roots of empty subtrees in a Merkle tree of depth 64."

Reply:
Makes sense, done!